src/annotator/integrations/html.js

```diff
@@ -24,6 +29,14 @@ export class HTMLIntegration {
     this.describe = describe;
 
     this._htmlMeta = new HTMLMetadata();
+
+    /**
+     * Whether to attempt to resize the document contents when {@link fitSideBySide}
+     * is called.
+     *
+     * Currently disabled by default.
+     */
+    this.sideBySideEnabled = false;
```

Comment:
This is the feature flag used to enable side-by-side mode for VitalSource EPUBs only initially. In future the plan is to enable it for HTML documents as well, once we've done more testing.